File: nysol_python/mcmd/methods/convTex/db/smp/mvsort.py
# ================================================
# コマンドマニュアル自動作成用サンプルコードデータ
# ================================================
db={}
db['name']='mvsort'

############################### IDATA
db['idatas']=[]

data={}
data['name']='dat1.csv'
data['text']='''
items1,items2
b a c,10 2
c c,2 5 3
e a a,1
'''
db['idatas'].append(data)

############################### SCRIPTS
db['scripts']=[]

script={}
script['title']='複数項目を並べる例'
script['comment']='''
``item1`` 項目を文字列降順に並べ、 ``item2`` 項目を数値昇順に並べる。
'''
script['sh_code']='''
mvsort vf=items1%r,items2%n i=dat1.csv o=rsl1.csv
'''
script['py_code']='''
nm.mvsort(vf="items1%r,items2%n", i="dat1.csv", o="rsl1.csv").run()
'''
script['odatas']='rsl1.csv'
db['scripts'].append(script)

File: olddoc/mcmd/en/examples/mvsort.rb
#!/usr/bin/env ruby
# coding: utf-8
require "./mkTex.rb"

File.open("dat1.csv","w"){|fpw| fpw.write(
<<'EOF'
items1,items2
b a c,10 2
c c,2 5 3
e a a,1
EOF
)}

############## Example 1
title="Sort multiple vectors"
comment=<<'EOF'
Sort \verb|item1| data series in ascending order and \verb|item2| in numerical ascending order.   
EOF
scp=<<'EOF'
more dat1.csv
mvsort vf=items1%r,items2%n i=dat1.csv o=rsl1.csv
more rsl1.csv
EOF
run(scp,title,comment)
